src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -80,27 +80,23 @@ private RootCommand SetupCommandLineAPI()
             return rootCommand;
         }
 
-        private void AnalyzeTemplate(FileInfo templateFilePath, FileInfo parametersFilePath, bool analyzeSingleTemplate = true)
+        private int AnalyzeTemplate(FileInfo templateFilePath, FileInfo parametersFilePath, bool analyzeSingleTemplate = true)
         {
             try
             {
-
                 string templateFileContents = File.ReadAllText(templateFilePath.FullName);
```

Comment:
Add one more check for when you pass a directory with the `analyze-template` command. Check for exists here similar to the check in `AnalyzeDirectory`